sylvia/tests/custom_query.rs

```diff
@@ -134,10 +134,48 @@ mod associated_type_interface {
     }
 }
 
+mod default_query_interface {
+    use cosmwasm_std::{Response, StdError, StdResult};
+    use sylvia::types::{ExecCtx, QueryCtx};
+    use sylvia::{contract, interface};
+
+    use crate::SomeResponse;
+
+    #[interface]
+    pub trait DefaultQueryInterface {
+        type Error: From<StdError>;
+
+        #[cfg(not(tarpaulin_include))]
+        #[msg(query)]
+        fn default_query(&self, ctx: QueryCtx) -> StdResult<SomeResponse>;
+
+        #[cfg(not(tarpaulin_include))]
+        #[msg(exec)]
+        fn default_exec(&self, ctx: ExecCtx) -> StdResult<Response>;
+    }
+
+    #[contract(module=super)]
+    #[sv::custom(query=MyQuery)]
+    impl DefaultQueryInterface for crate::MyContract {
+        type Error = StdError;
+
+        #[msg(query)]
+        fn default_query(&self, _ctx: QueryCtx) -> StdResult<SomeResponse> {
```

Comment:
We need an additional test here.

We need a query variant here with the custom query - taking the `QueryCtx<MyQuery>` context.  Then, in one of the tests, we need to create a multitest test with a Custom module queried by this contract, making sure that the querying works correctly here.